### DISCUSSION TITLE
A "Calendar Wanted" help list?

### DISCUSSION THREAD

**Discussion by ghost** [vacanza/holidays#959] [Ideas]
This package is extremely helpful! 

I noticed Qatar, Libya and some other states in the Islamic world are missing. Which led to me compare holidays against the UN member nations list. Could you maintain a list of "UN members needing calendar?" Instead of being requesting one by one, it would call attention to where people can help expand what is BY FAR the best package in python for handling holiday calendars.

**Comment by arkid15r** [comment]
Hi @jb2987,
thanks for your effort in improving this package!

We've made significant progress in terms of number of supported country codes recently (86 in v0.16 vs 119 in v0.20-beta) and going to continue extending this list! 

I like your idea of using UN countries list to attract more contributions. I'm going to create an issue based on your suggestion early next week.

**Reply by ghost** [reply]
Happy to help! There's great work being done here.

**Comment by mborsetti** [comment]
FYI, the full list of country codes is at https://en.wikipedia.org/wiki/ISO_3166-1#Officially_assigned_code_elements.

There's 248 of them.

**Comment by arkid15r** [comment]
The UN based list of new countries can be found [here](https://github.com/dr-prodigy/python-holidays/issues/1141). 
The wider country code support discussion is still open.
